packages/next/src/build/babel/plugins/react-loadable-plugin.ts

```diff
@@ -120,7 +120,7 @@ export default function ({
 
           properties.forEach((property) => {
             const key: any = property.get('key')
-            propertiesMap[key.node.name] = property
+            propertiesMap[key.node.name] = property as any
```

Comment:
I can remove this locally and still build and `types-and-precompiled`. Why is this needed?

Reply:
it was failing during the upgration, going to revert it since it's not failing anymore for me as well